lib/pg/execute.js

```diff
@@ -84,7 +84,13 @@ const executeInsertCQN = async (model, dbc, cqn, user, locale, txTimestamp) => {
 
   let results = await Promise.all(resultPromises)
   results = flattenArray(results)
-  results = results.map((result) => remapColumnNames(model.definitions[cqn.INSERT.into], result))
+  let entity
+  if (cqn.INSERT.into && cqn.INSERT.into.ref && cqn.INSERT.into.ref[0]) {
```

Comment:
let's make use of optional chaining here:
`const entity = cqn.INSERT?.into?.ref?[0] ? cqn.INSERT.into.ref[0] : cqn.INSERT.into`